crypto/pem/pem_passwd.c

```diff
@@ -0,0 +1,304 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#include <assert.h>
+#include <ctype.h>
+#include <stdio.h>
+#include <string.h>
+
+#include <openssl/buf.h>
+#include <openssl/err.h>
+#include <openssl/evp.h>
+#include <openssl/pem.h>
+#include <openssl/x509.h>
+
+#include "internal.h"
+#include "../internal.h"
+
+# include <signal.h>
```

Comment:
Minor bit but you can correct the white space (remove it) for these preprocessor macros like`# include`? I know this is an OpenSSL thing but it doesn’t much our style and is inconsistent throughout this file.